core/src/main/java/co/aikar/commands/annotation/CatchUnknown.java

```diff
@@ -26,6 +26,13 @@
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
+/**
+ * Gets all subcommands which are unknown at the time. This means any arguments which are unresolved and have no
```

Comment:
wording is just super confusing here.

this is to mark a method that should be executed anytime an unknown command is encountered.